src/content/docs/query-your-data/explore-query-data/use-charts/on-demand-forecasting.mdx

```diff
@@ -0,0 +1,129 @@
+Follow these steps to create a forecast on a line or area chart in New Relic:
+
+<Steps>
+
+  <Step>
+    ### Access a line or area chart [#access-a-line-chart]
+
+    1. Go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities).
+    2. Choose a line or area chart from <DNT>**Dashboards**</DNT>, or open it from <DNT>**Metrics and events**</DNT>.
+  </Step>
+
+  <Step>
+    ### Make a forecast [#make-a-forecast]
+
+  In the upper-right corner of the chart, select <DNT>**Make a forecast**</DNT> from the <DNT>**Options**</DNT> menu. 
+  The forecasted chart opens in <DNT>**Data explorer**</DNT>. 
+  The forecast range is automatically set to 20% of the query window, and highlighted in gray. The graph appears in dotted lines within the forecast range. 
+
+  The chart query is pre-populated with the `FORECAST` function. you can [refine the query](#customize-forecastsing-range) as per your requirements and run it to get the updated graph.
+
+  </Step>
+
+  <Step>
+    ### Add to dashboard [#add-to-dashboard]
+
+    1. Click <DNT>**Add to dashboard**</DNT>.
+    2. Enter a widget name.
+    3. Search and slect a dashboard in which you want to add the widget, or create a new dashboard for it.
+
+    You can now view the forecasted chart on the selected dashboard.
+
+  </Step>
+  
+</Steps>
+
+## Default forecasting range [#default-forecasting-range]
+The default forecast range in New Relic provides a baseline for projecting future trends based on historical data. 
+This range is automatically set to ensure users receive meaningful insights without a manual NRQL configuration. 
+Understanding these default settings helps users make informed decisions about when and how to customize their forecasts for more precise predictions.
+
+### Default values [#default-values]
+When you create a forecast in New Relic, the system automatically appends the `FORECAST` function to the existing chart query, and sets the following default values:
+* **Forecast range**: The forecast covers 20% of the total time specified in your query window, allowing you to see a short-term projection based on current trends.
+* **Historical data**: The system analyzes historical data that is twice the length of your query window, providing a comprehensive dataset to enhance the accuracy of the forecast.
+
+
+### Customize the forecasting range [#customize-forecastsing-range]
+You can customize a chart forecast through NRQL query to adjust the projection window and refine the accuracy as per your requirements. 
+Refer to the [NRQL documentation](/docs/nrql/get-started/introduction-nrql-new-relics-query-language) to get familiar with the NRAQL query. 
+
+When you create a forecast of a chart, the revised NRQL query appears with the `FORECAST` function, as in the following example:
+
+```sql
+FROM NrAiIncidentDebug SELECT count(*)  WHERE environment like 'production'
+TIMESERIES 1 hour facet event since 1 day ago FORECAST
```

Comment:
I'd capitalize the keywords, i.e. FROM NrAiIncidentDebug SELECT count(*)  WHERE environment LIKE 'production'
TIMESERIES 1 hour FACET event SINCE 1 day ago FORECAST